test/countries/test_spain.py

```diff
@@ -265,6 +206,115 @@ def test_variable_days_in_2022(self):
                     % (date(2022, *fest_date), prov, ", ".join(fest_provs)),
                 )
 
+    def test_variable_days_in_2023(self):
+        province_days = {
+            (1, 2): [
```

Comment:
Optional: if you're in a mood to go an extra mile it'd be great to have months replaced with their constants from `constants.py` (e.g., (1, 2) -> (JAN, 2), (2, 21) -> (FEB, 21)

Reply:
Done